styles.js

```diff
@@ -1,6 +1,28 @@
 import { createGlobalStyle } from "styled-components";
 
 export default createGlobalStyle`
+
+@font-face {
+    font-family: 'Roboto Slab';
+    src: url('/fonts/Roboto_Slab/RobotoSlab-Regular.ttf') format('truetype');
+    font-weight: normal;
+    font-style: normal;
+  }
+
+  @font-face {
+    font-family: 'Lusitana';
+    src: url('/fonts/Lusitana/Lusitana-Regular.ttf') format('truetype');
+    font-weight: normal;
+    font-style: normal;
+  }
+
+  @font-face {
+    font-family: 'Biryani';
+    src: url('/fonts/Biryani/Biryani-Regular.ttf') format('truetype');
+    font-weight: normal;
+    font-style: normal;
+  }
+
```

Comment:
You don't need to install fonts. Next comes with all fonts included. (https://nextjs.org/docs/pages/building-your-application/optimizing/fonts)